pkg/manifests/manifests.go

```diff
@@ -674,6 +674,19 @@ func ClusterRoleManifest() *rbacv1.ClusterRole {
 					"nodes",
 				},
 			},
```

Comment:
What's the motivation behind this? Shoudn't this be already implicit by https://github.com/kubernetes/kubernetes/blob/a3ccea9d8743f2ff82e41b6c2af6dc2c41dc7b10/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L231 and https://github.com/kubernetes/kubernetes/blob/a3ccea9d8743f2ff82e41b6c2af6dc2c41dc7b10/plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go#L161

Reply:
We don't run the bootstrap policy in actuator e2e. Thus, we need to create the clusterrole explicitly.